app/app.go

```diff
@@ -133,12 +143,12 @@ var (
 		stakingtypes.NotBondedPoolName: {authtypes.Burner, authtypes.Staking},
 		govtypes.ModuleName:            {authtypes.Burner},
 		ibctransfertypes.ModuleName:    {authtypes.Minter, authtypes.Burner},
-		oracletypes.ModuleName:         nil,
+		oracletypes.ModuleName:         nil, // TODO:??
```

Comment:
will need perms for rewarding voters, but we need to design the incentive model first.